hummingbird/templates/add_user.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
```

Comment:
nit summary:
- Indent content inside `body_block` and nested `if` blocks
- Add newline to end of file